lib/preparser.ts

```diff
@@ -53,6 +53,12 @@ export async function preparseArgs(argv: string[]): Promise<string[]> {
 		if (extractBooleanFlag(cmdSlice, '--debug')) {
 			process.env.DEBUG = '1';
 		}
+		// support global --v-next flag
```

Comment:
We now have a `--v-next` that can be used anywhere, similar to `--debug`, which makes new major version testing much more convenient.